lib/pagination.js

```diff
@@ -26,7 +29,9 @@ function pagination(base, posts, options = {}) {
     if (urlCache.has(i)) return urlCache.get(i);
 
     let url;
-    if (!explicitPaging) {
+    if (renameLast && explicitPaging && i === total) {
+      url = base + format(_format, i).replace(i, localizedLast);
```

Comment:
`.replace(i, localizedLast)`

this still a potential problem, it may replace other content in the url.

```suggestion
      url = base + _format.replace('%d', localizedLast);
```